modules/compiler/targets/riscv/test/lit/passes/link-builtins-memcpy.ll

```diff
@@ -0,0 +1,52 @@
+; Copyright (C) Codeplay Software Limited
+;
+; Licensed under the Apache License, Version 2.0 (the "License") with LLVM
+; Exceptions; you may not use this file except in compliance with the License.
+; You may obtain a copy of the License at
+;
+;     https://github.com/codeplaysoftware/oneapi-construction-kit/blob/main/LICENSE.txt
+;
+; Unless required by applicable law or agreed to in writing, software
+; distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+; WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+; License for the specific language governing permissions and limitations
+; under the License.
+;
+; SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+; RUN: muxc --device "%riscv_device" --passes link-builtins,verify -S %s | FileCheck %s
+
+target triple = "riscv64-unknown-unknown-elf"
+target datalayout = "e-m:e-p:64:64-i64:64-i128:128-n32:64-S128"
+
+; CHECK-NOT: call void @llvm.memcpy
+; CHECK: define linkonce dso_local spir_func noundef ptr @memmove(
```

Comment:
This looks like a wrong test, I think you wanted to check in this file that `memcpy` works, not that `memmove` works?

Reply:
ah yes quite right